css/templates/e-commerce/page.css

```diff
@@ -557,3 +557,32 @@ amp-selector [option][selected] {
     margin-left: calc(1/12 * 100%);
   }
 }
+
+
+/* CART */
+.commerce-cart-icon {
+  color: var(--color-font-accent);
+}
+
+.commerce-cart-total {
+  border-top: 1px solid var(--color-separator);
+  border-bottom: 1px solid var(--color-separator);
+}
+
+@media (--breakpoint-md) {
+  .commerce-cart-item {
+    border-bottom: 1px solid var(--color-separator);
+  }
+
+  .commerce-cart-button {
+    text-align: left;
```

Comment:
Align button to left on wider screens. I could avoid this div but it'd mean having to change the layout of the parent container to flex. The downside of this approach is the button width wouldn't be determined by the button text (unless I add an empty div around the button), which puts us in the same place (in terms of markup).